torchvision/prototype/models/_api.py

```diff
@@ -34,7 +35,7 @@ class Weights:
     meta: Dict[str, Any]
 
 
-class WeightsEnum(Enum):
+class WeightsEnum(StrEnum):
```

Comment:
I'm about to move the Models prototype to the main area. We can't depend on an `StrEnum` that is on the prototype area here.

Reply:
True. Any reservations of using `StrEnum` also to the main area? I would put it into a private utils module like `torchvision._utils`.